cdmtaskservice/condor/run_job.sh

```diff
@@ -33,35 +42,37 @@ for var in "${EXPECTED_VARS[@]}"; do
   if [[ -v "$var" ]]; then
     value="${!var}"
     echo "$var=$value"
+    # escape single quotes safely for inclusion in shell file
     safe_value=${value//\'/\'\\\'\'}
     echo "export $var='$safe_value'" >> "$ENV_SOURCE_FILE"
   fi
 done
 
+if [[ -n "${ADDITIONAL_PATH:-}" ]]; then
+  safe_addl_path=${ADDITIONAL_PATH//\'/\'\\\'\'}
+  echo "export PATH='$safe_addl_path':\$PATH" >> "$ENV_SOURCE_FILE"
```

Comment:
I didn't notice any `ADDITIONAL_PATH` validations - can ADDITIONAL_PATH contain spaces or special characters - thus a potential injection vulnerability here?

Reply:
There isn't any validation, but it's only settable by service admins so I didn't worry about it

That being said, I've added a TODO locally to add validation in the future.